project_partner_name/README.rst

```diff
@@ -0,0 +1,49 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=========================
+Project Partner Name
+=========================
+
+This module extends the functionality of Project:
+
+* Makes "Customer" mandatory field.
+* Modifies the name_get adding the customer reference to the project name.
+* Modifies the name_search to search also by customer reference linked to project.
```

Comment:
does it solve the 2 pending bugs:

* name_get of analytic account should be added as well
* when in task / project field: the name of the first customer is repeated in all lines